src/multiplex/client.rs

```diff
@@ -244,24 +242,19 @@ where
                 })) => {
                     let id = transport.as_mut().assign_tag(&mut req);
 
-                    #[cfg(feature = "tracing")]
                     let guard = _span.enter();
-                    #[cfg(feature = "tracing")]
                     tracing::event!(Level::TRACE, "request received by worker; sending to Sink");
 
                     transport
                         .as_mut()
                         .start_send(req)
                         .map_err(Error::from_sink_error)?;
-                    #[cfg(feature = "tracing")]
                     tracing::event!(Level::TRACE, "request sent");
```

Comment:
TIOLI: could be more concise as 
```suggestion
                    tracing::trace!("request sent");
```